smart-contracts/hardhat.config.js

```diff
@@ -33,7 +33,7 @@ const { getHardhatNetwork } = require('./helpers/network')
 const settings = {
   optimizer: {
     enabled: true,
-    runs: 200,
+    runs: 90,
```

Comment:
😨

Reply:
Oh right this can be brought back to 200. Will change back